### DISCUSSION TITLE
Dubbio corretta installazione HA

### DISCUSSION THREAD

**Discussion by Tempesta72** [indomus/forum#2236] [HUB: Home Assistant]
Buongiorno a tutti, perdonatemi abbiate pazienza. Grazie alle guide indomus ho installato HA core con docker su raspberrypi model 3 e sembra funzionare tutto correttamente. Il mio dubbio (e al riguardo forse ho già incrociate qualche discussione ma comunque la situa non mi è chiara) è relativo al percorso di installazione. In buona sostanza HA non si trova sotto pi ma, più precisamente in /home/homeassistant/.homeassistant
E' il caso di spostare il tutto sotto pi ? 
In ssh con putty mi collego e con il comando sudo non ci sono problemi ma, con visual studio code non riesco a collegarmi (semplicemente mi dice che non riesce a stabilire una connessione con l'indirizzo del raspberry). Può essere un problema legato alla posizione di installazione e quindi di permessi ?
grazie a tutti in anticipo e, speriamo, buon 2023 a tutti

**Comment by snakuzzo** [comment]
La directory di installazione presente nelle guide è solo un esempio...puoi metterla dove ti pare. Quindi, per rispondere alla tua prima domanda, non devi spostare niente...va bene così.
Quanto a VS Code, se l'errore è quello che dici, hai sbagliato ad impostare qualcosa.

**Comment by Tempesta72** [comment]
perfetto ti ringrazio.
per quanto riguarda VSC è strano, con putty o filezilla non ho problemi, ma con lui si...., indagherò, grazie ancora

**Reply by snakuzzo** [reply]
Per favore fai reply sotto l'ultimo messaggio...non come hai fatto ora ... hai creato un nuovo thread.
Comunque `sudo` e VS Code non c'entrano nulla l'uno con l'altro.

**Reply by Tempesta72** [reply]
sorry.
nono, capisco che `sudo` e VS code non c'entrano nulla, dovrebbe essere semplicemente un problema di comunicazione in ssh
comunque grazie ancora e scusa per il thread
